pkg/apisix/resource_test.go

```diff
@@ -86,8 +86,11 @@ func TestItemConvertUpstream(t *testing.T) {
 	ups, err := ite.upstream()
 	assert.Nil(t, err)
 	assert.Len(t, ups.Nodes, 2)
-	assert.Equal(t, ups.Nodes[0], v1.UpstreamNode{Host: "httpbin.org", Port: 80, Weight: 1})
-	assert.Equal(t, ups.Nodes[1], v1.UpstreamNode{Host: "foo.com", Port: 8080, Weight: 2})
+	if ups.Nodes[0].Host == "foo.com" {
+		ups.Nodes[0], ups.Nodes[1] = ups.Nodes[1], ups.Nodes[0]
```

Comment:
what's this?

Reply:
The position values of 1 and 0 may be different.